src/api/create-repository/controller.js

```diff
@@ -33,14 +34,26 @@ const createRepositoryController = {
 
     request.logger.info(`Creating repository: ${repositoryName}`)
 
-    await createRepositoryStatus(
+    await initCreationStatus(
       request.db,
-      gitHubOrg,
+      org,
+      creations.repository,
       repositoryName,
-      payload,
-      team.github
+      config.get('workflows.createRepository'),
+      undefined,
```

Comment:
😱 this is super prone to bugs 😱

I'd switch this to pass an object as the functions arg